python/lsst/ts/mtdome/mock_llc/mock_motion/elevation_motion.py

```diff
@@ -157,35 +157,20 @@ def get_position_velocity_and_motion_state(
         position = utils.angle_wrap_nonnegative(math.degrees(position)).rad
         return position, velocity, motion_state
 
-    def stop(self, start_tai: float) -> float:
-        """Stops the current motion.
+    def exit_fault(self, start_tai: float) -> float:
+        """Clear the fault state.
```

Comment:
Is this still needed? I hope you can use the version in the parent class, but if not, please explain why in the doc string.

Reply:
I added a comment explaining why it is still needed.